### ISSUE TITLE
lots of classifier runs don't show up

### ISSUE THREAD

**Issue by superawesome** [superawesome/classicalc#12]
Sometimes clubs don't report match data, only classifier data. Consequently those runs never show up anywhere on uspsa.org except that members' classifier profile page, and then only once they've been officially ranked (monthly around the 15th).

Potential fixes would be:

1) Allow people to enter their own data. Requires some form of login, and perhaps even validation about which USPSA ID is theirs. Sadpanda.

2) Allow people to enter a USPSA ID to have tracked. We would then periodically scrape that ID's classifier page and show the union of that data plus the classifier_runs data... if possible. Might be hard to figure out which things are duplicates. This could work, but obviously gives up speed- you won't see your data until the USPSA officially ranks it.

3) Pull data from Practiscore and union that with the classifier_runs data. Probably covers many of the problematic cases, but perhaps not all of them. On the other hand, it's probably faster than waiting for the official monthly processing that #2 would require. There's another issue to track this already.

4) Attempt to scrape all of the classifier profile pages. Extremely tedious because there's no obvious way to enumerate all valid ID's, but thorough. Doesn't solve the immediacy problem. I think I prefer #2 over this.
